boards/sim/sim/sim/configs/segger/defconfig

```diff
@@ -62,6 +62,7 @@ CONFIG_SCHED_HAVE_PARENT=y
 CONFIG_SCHED_HPWORK=y
 CONFIG_SCHED_INSTRUMENTATION=y
 CONFIG_SCHED_INSTRUMENTATION_FILTER=y
+CONFIG_SCHED_INSTRUMENTATION_SYSCALL=y
```

Comment:
let's modify https://github.com/apache/incubator-nuttx/blob/master/tools/ci/testlist/sim-02.dat to remove segger from macOS build.

Reply:
Done